src/ImageRunModal.jsx

```diff
@@ -228,6 +230,9 @@ export class ImageRunModal extends React.Component {
             const memorySize = this.state.memory * (1024 ** units[this.state.memoryUnit].base1024Exponent);
             createConfig.memory = memorySize.toString();
         }
+        if (this.state.cpuSharesConfigure && this.state.cpuShares) {
```

Comment:
This means that a value of 0 would be ignored. Is that intended? 0 may be the internal default, but not sure if we should hardcode that knowledge. Why not just actually set 0 if the user specified it?